dev/docgen/mapDir.ts

```diff
@@ -35,18 +35,31 @@ export const mapDir = (
                         transformedOutputPath
                     )
                 }
-                return [transformedOutputPath, transformedContents]
+                return [
+                    transformedOutputPath,
+                    transformedContents,
+                    repoRelativePath
+                ]
             })
     )
     for (const target of options.targets) {
+        const sourceMapData: Record<string, string> = {}
+        const sourceMapPath = join(target, ".sourceMap.json")
         rmSync(target, { recursive: true, force: true })
```

Comment:
Don't need to change it in this PR but the `recursive` and `force` options are only needed for directories.